app/Models/UserModel.php

```diff
@@ -30,4 +32,33 @@ protected function hashPassword(array $data)
 
         return $data;
     }
+
+    public function updatePassword($userId, $newPassword, $isConfirmed)
+    {
+        try {
+            $this->db->transStart();
```

Comment:
```suggestion
            $this->db->transException(true)->transStart();
```
Make this change please for the enable that the db throws the exceptions